diff -urN TleenX2-20031002.orig/src/Makefile.am TleenX2-20031002/src/Makefile.am
--- TleenX2-20031002.orig/src/Makefile.am	2003-09-19 16:55:57.000000000 +0200
+++ TleenX2-20031002/src/Makefile.am	2003-11-09 00:34:48.000000000 +0100
@@ -29,6 +29,6 @@
 	gg.c gg.h
 
 tleenx2_LDADD = @PACKAGE_LIBS@  ../jabby/libjabby.a ../iksemel/libiksemel.a \
-	/usr/X11R6/lib/libXss.a
+	-lXss
 tleenx2_LDFLAGS = `pkg-config --libs gthread-2.0`
 tleenx2_CFLAGS = -I../jabby -I../iksemel
