diff -urN tleenx2.orig/src/utils.c tleenx2/src/utils.c
--- tleenx2.orig/src/utils.c	Thu May  8 19:54:02 2003
+++ tleenx2/src/utils.c	Thu May  8 19:55:23 2003
@@ -295,10 +295,10 @@
 {
   switch(status)
   {
-    case 0:
+/*    case 0:
       return create_pixmap(window1,"noauth.gif");
     case 1:
-      return create_pixmap(window1,"actions_o.png");
+      return create_pixmap(window1,"actions_o.png");*/
     case TLEN_STATUS_UNAVAILABLE:
       return create_pixmap(window1,"offline.png");
     case TLEN_STATUS_AWAY:
