diff -Nur TleenX2-20031002.orig/Makefile.am TleenX2-20031002/Makefile.am
--- TleenX2-20031002.orig/Makefile.am	Sat Nov  8 20:58:49 2003
+++ TleenX2-20031002/Makefile.am	Sat Nov  8 21:04:23 2003
@@ -18,6 +18,17 @@
 	  done \
 	fi
 
+install-man-local:
+	@$(NORMAL_INSTALL)
+	if test -f $(srcdir)/*.1; then \
+	  $(mkinstalldirs) $(DESTDIR)$(mandir)/man1; \
+	  for mans in $(srcdir)/*.1; do \
+	    if test -f $$mans; then \
+	      $(INSTALL_DATA) $$mans $(DESTDIR)$(mandir)/man1; \
+	    fi \
+	  done \
+	fi
+
 dist-hook:
 	if test -d pixmaps; then \
 	  mkdir $(distdir)/pixmaps; \
@@ -27,4 +38,11 @@
 	    fi \
 	  done \
 	fi
-
+	  if test -f *.1; then \
+	    mkdir $(distdir)/man/man1; \
+	    for mans in $(srcdir)/*.1; do \
+	      if test -f $$mans; then \
+		cp -p $$mans $(distdir)/man/man1; \
+	      fi \
+	    done \
+	fi
diff -Nur TleenX2-20031002.orig/Makefile.in TleenX2-20031002/Makefile.in
--- TleenX2-20031002.orig/Makefile.in	Sat Nov  8 20:58:49 2003
+++ TleenX2-20031002/Makefile.in	Sat Nov  8 21:08:38 2003
@@ -480,7 +480,7 @@
 uninstall: uninstall-recursive
 
 install-am: all-am
-	@$(MAKE) $(AM_MAKEFLAGS) install-exec-am install-data-am
+	@$(MAKE) $(AM_MAKEFLAGS) install-exec-am install-data-am install-man
 
 installcheck: installcheck-recursive
 install-strip:
@@ -514,13 +514,13 @@
 
 info-am:
 
-install-data-am: install-data-local install-man
+install-data-am: install-data-local  
 
 install-exec-am:
 
 install-info: install-info-recursive
 
-install-man: install-man1
+install-man: install-man1 install-man-local
 
 installcheck-am:
 
@@ -545,7 +545,7 @@
 
 uninstall-info: uninstall-info-recursive
 
-uninstall-man: uninstall-man1
+uninstall-man: uninstall-man1 uninstall-man-local
 
 .PHONY: $(RECURSIVE_TARGETS) CTAGS GTAGS all all-am check check-am clean \
 	clean-generic clean-recursive ctags ctags-recursive dist \
@@ -556,14 +556,14 @@
 	install-data-am install-data-local install-data-recursive \
 	install-exec install-exec-am install-exec-recursive \
 	install-info install-info-am install-info-recursive install-man \
-	install-man1 install-recursive install-strip installcheck \
+	install-man1 install-man-local install-recursive install-strip installcheck \
 	installcheck-am installdirs installdirs-am \
 	installdirs-recursive maintainer-clean maintainer-clean-generic \
 	maintainer-clean-recursive mostlyclean mostlyclean-generic \
 	mostlyclean-recursive pdf pdf-am pdf-recursive ps ps-am \
 	ps-recursive tags tags-recursive uninstall uninstall-am \
 	uninstall-info-am uninstall-info-recursive uninstall-man \
-	uninstall-man1 uninstall-recursive
+	uninstall-man1 uninstall-man-local uninstall-recursive
 
 
 install-data-local:
@@ -577,6 +577,17 @@
 	  done \
 	fi
 
+install-man-local:
+	@$(NORMAL_INSTALL)
+	if test -f $(srcdir)/*.1; then \
+	  $(mkinstalldirs) $(DESTDIR)$(mandir)/man1; \
+	  for mans in $(srcdir)/*.1; do \
+	    if test -f $$mans; then \
+	      $(INSTALL_DATA) $$mans $(DESTDIR)$(mandir)/man1; \
+	    fi \
+	  done \
+	fi
+
 dist-hook:
 	if test -d pixmaps; then \
 	  mkdir $(distdir)/pixmaps; \
@@ -586,6 +597,16 @@
 	    fi \
 	  done \
 	fi
+
+	if test -f *.1; then \
+	  mkdir $(distdir)/man/man1; \
+	  for mans in $(srcdir)/*.1; do \
+	    if test -f $$mans; then \
+	      cp -p $$mans $(distdir)/man/man1; \
+	    fi \
+	  done \
+	fi
+
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
 .NOEXPORT:
